app/frontend/src/views/admin/Submission.vue

```diff
@@ -93,6 +113,12 @@ export default {
     ...mapActions('form', ['getForm']),
     refreshNotes() {
       this.$refs.notesPanel.getNotes();
+    },
+    operationTypeDisplayName(operationType){
+      const obj =  this.operationTypes.filter( function(el){
+        return el.type == operationType;
+      });
```

Comment:
Can just be
`this.operationTypes.filter(type => type === operationType);`
or better yet
`this.operationTypes.find(type => type === operationType);` then you don't need the [0]

Reply:
nice. i'll have to remember this one.